### ISSUE TITLE
Sound stops working when plugging in headphone

### ISSUE THREAD

**Issue by duckunix** [yusefnapora/pixelbook-linux#14]
Ubuntu 19.04, either Gnome or i3-wm.

Sound works great on the speakers. Nice and clear and loud.  Plug in headphones in the jack, and no more sound, and anything to do with sounds seems to be blocking, ie videos will not play, etc.

Only way to clear up from what I have tried is to reboot.

**Comment by ranalabs** [comment]
I can confirm this happens to me as well. Did you ever find a solution? Thanks!

**Comment by duckunix** [comment]
Nope.  The only way I can get sound working is to use Bluetooth.  Even having a USB audio card does not work.  I know the USB works as I have been using it for years under Linux, and it works on another system currently.

**Comment by ranalabs** [comment]
Same here, bluetooth works fine. I can live with that I suppose.
